### PR TITLE
Fix site based on recommendations

### PR DESCRIPTION
**Pull request by Cerdtruir**
Fix site based on recommendations from previous pr and other feedback

Many small changes

demo: http://htmlpreview.github.io/?https://github.com/Cerdtruir/WiFi4Kids/blob/fix_site_based_on_recommendations/index.html

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -200,3 +210,7 @@ footer {
   align-items: center;
   justify-content: center;
 }
+
+#loadOverlay {
+  display: none;
+}
```
Comment (ikraamg) [inline style.css:216]: Nice trick!

File: index.html [hunk 2]
```diff
@@ -8,26 +8,38 @@
...
+    <div
+      id="loadOverlay"
+      style="
+        background-color: #333;
+        position: absolute;
+        top: 0px;
+        left: 0px;
+        width: 100%;
+        height: 100%;
+        z-index: 2000;
```
Comment (ikraamg) [inline index.html:20]: To make sure it's on top 😂

**Review by ikraamg — APPROVED** [review]
This looks good 👍

File: index.html (hunk 2 above)
Comment (Cerdtruir) [reply index.html:20]: i googled for this, html was loading before css

**Pull request merged**